content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2215,6 +2245,12 @@ The _name_ of the `fields` argument on the `@@index` attribute can be omitted:
 @@index(_ fields: FieldReference[], name: String?)
 ```
 
+Starting with 2.29 if the `namedConstraints` preview flag is enabled the signature changes. The old name argument will still be accepted though to prevent a breaking change.
```

Comment:
```suggestion
As of 2.29.0, if the `namedConstraints` preview flag is enabled the signature changes. 

>**Note**: The old `name` argument will still be accepted to prevent a breaking change.
```